packages/hash/eslint-config/block.js

```diff
@@ -4,16 +4,17 @@ module.exports = {
     jsx: true,
     useJSXTextNode: true,
   },
-  plugins: ["@typescript-eslint", "react-hooks", "react"],
+  plugins: ["@typescript-eslint", "react-hooks", "react", "unicorn"],
```

Comment:
Checks `kebab-case` file names in blocks (to be consistent with new code in the `fronted` package)